AdvLoggerPkg/Library/AdvancedLoggerLib/MmCore/AdvancedLoggerLib.c

```diff
@@ -7,7 +7,7 @@
 **/
 
 #include <PiDxe.h>
-
+#include <PiMm.h>

```

Comment:
Can we split this up into a REBASE & FF PR with the actual lib changes and the unit test separated?

Reply:
https://github.com/microsoft/mu_plus/pull/582

Reply:
Merged.